code/modules/medical/pathology/pathogen_symptoms.dm

```diff
@@ -2013,24 +2013,24 @@ datum/pathogeneffects/malevolent/radiation
 		switch (origin.stage)
 			if (1 to 3)
 				if (prob(5 * origin.stage + 3))
-					M.changeStatus("radiation", origin.stage)
+					M.take_radiation_dose(origin.stage)
 					boutput(M,"<span class='alert'>You feel sick.</span>")
 			if (4)
 				if (prob(13))
-					M.changeStatus("radiation", 3 SECONDS)
+					M.take_radiation_dose(3 SECONDS)
 					boutput(M,"<span class='alert'>You feel very sick!</span>")
 				else if (prob(26))
-					M.changeStatus("radiation", 2 SECONDS)
+					M.take_radiation_dose(2 SECONDS)
 					boutput(M,"<span class='alert'>You feel sick.</span>")
 			if (5)
 				if (prob(15))
-					M.changeStatus("radiation", rand(2,4) SECONDS)
+					M.take_radiation_dose(rand(2,4) SECONDS)
 					boutput(M,"<span class='alert'>You feel extremely sick!!</span>")
 				else if (prob(20))
-					M.changeStatus("radiation", 3 SECONDS)
+					M.take_radiation_dose(3 SECONDS)
 					boutput(M,"<span class='alert'>You feel very sick!</span>")
 				else if (prob(40))
-					M.changeStatus("radiation", 2 SECONDS)
+					M.take_radiation_dose(2 SECONDS)
 					boutput(M,"<span class='alert'>You feel sick.</span>")
```

Comment:
on the one hand, patho is disabled, and probably forever, so this doesn't really matter
on the other hand, dear lord.

Reply:
I'mma leave it in the hope that someone tries out patho one day and gets massively irradiated for their troubles :satanfrog: